mod_app/admin/film_admin.py

```diff
@@ -4,21 +4,22 @@
 from django.template.defaultfilters import truncatechars_html
 from django.utils.html import format_html, mark_safe
 
-from mod_app.admin.link_admin import (
-    CardImageInline,
-    DrawingInline,
-    OtherLinkInline,
-    PostcardInline,
-    PosterInline,
-    PressBookInline,
-    ProgrammeInline,
-    PublicVisualInfluenceInline,
-    PublicityInline,
-    ScriptInline,
-    SourceInline,
-    StillInline,
-    VideoInline,
-)
+# from mod_app.admin.link_admin import (
+#     CardImageInline,
```

Comment:
we'll want to make sure we still import cardimage, otherlink, source and video even if we exclude the others